src/reason-parser/reason_pprint_ast.ml

```diff
@@ -7576,29 +7576,42 @@ let printer = object(self:'self)
     | Pmod_constraint _
     | Pmod_structure _ -> self#simple_module_expr x
 
-
   method structure structureItems =
+    (* We don't have any way to know if an extension is placed at the top level by the parsetree
```

Comment:
I thought we supported extensions at the toplevel with `%`, and that `%%bs.raw` was specifically a bucklescript thing.

Reply:
It is a BuckleScript thing, but we support both cases. Would be a nice idea to unify all of these cases with Melange. `%bs.raw` -> `bs.raw_js_expr` and `%%bs.raw` -> `bs.raw_js` or something like this.

Reply:
I don't get what needs to be unified?

Reply:
We could avoid having `%` and `%%` at the same time and support only `%`